tests/integ/modin/series/test_aggregate.py

```diff
@@ -344,19 +342,6 @@ def test_skew_series():
     assert round(snow_df["A"].skew(), 4) == round(native_df["A"].skew(), 4)
 
 
-@sql_count_checker(query_count=0)
-def test_named_agg_not_supported_function(native_series):
```

Comment:
named aggreation for series doesn't work in Snowpark pandas or in pandas, so we don't need to test this case: https://github.com/snowflakedb/snowpark-python/blob/f24efc4e651a4cbecda260b3efd4a136142dee8b/src/snowflake/snowpark/modin/pandas/utils.py#L553

Reply:
Do we want to keep a negative test around to ensure this remains the case?

Reply:
There's already a test case here: https://github.com/snowflakedb/snowpark-python/blob/ed7c83a7bda63468c9e551488674cbc8f231c12b/tests/integ/modin/series/test_aggregate.py#L239